### ISSUE TITLE
Docker Desktop window captures CTRL-R globally when open

### ISSUE THREAD

**Issue by douglaswth** [docker/for-win#12495]
- [x] I have tried with the latest version of Docker Desktop
  - [x] I have tried disabling enabled experimental features
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: 103E72D4-07BB-4D65-B4C3-903CB774E995/20220113231603

### Actual behavior

When the Docker Desktop window is open, it seems to register a global keyboard shortcut (`CTRL-R`) to restart Docker that will trigger even if the window is not focused. So, if I am in a terminal and type `CTRL-R` to search through my shell history or if I am in a web browser and want to refresh, instead of doing what I thought would happen, I am instead prompted with a Docker restart:

![image](https://user-images.githubusercontent.com/77890/149424667-17119218-f040-4d22-96cc-950bafb993e5.png)

This seems to be new behavior after upgrading from 4.3.2 to 4.4.2.

### Expected behavior

The expected behavior would be for the `CTRL-R` shortcut to only have this behavior when the Docker Desktop window is focused so that the shortcut would work as expected in other apps.

### Information

  - Windows Version: Windows 10 21H1
  - Docker Desktop Version: 4.4.2
  - WSL2 or Hyper-V backend?  WSL2
  - Are you running inside a virtualized Windows e.g. on a cloud server or a VM: No

### Output of `& "C:\Program Files\Docker\Docker\resources\com.docker.diagnose.exe" check`

```
[013:15:30:58.683][I] ipc.NewClient: 09a24884-com.docker.diagnose -> \\.\pipe\dockerBackendV2 com.docker.service.exe
[goroutine 1 [running, locked to thread]:
[win/backend/pkg/service.NewClientForPath(...)
[       win/backend/pkg/service/service.go:31
[win/backend/pkg/service.NewClient({0xa2d772, 0x13})
[       win/backend/pkg/service/service.go:25 +0xc5
[common/pkg/diagkit/gather/diagnose.init()
[       common/pkg/diagkit/gather/diagnose/ipc_windows.go:23 +0x118
Starting diagnostics

[PASS] DD0027: is there available disk space on the host?
[PASS] DD0028: is there available VM disk space?
[PASS] DD0031: does the Docker API work?
[PASS] DD0004: is the Docker engine running?
[PASS] DD0011: are the LinuxKit services running?
[PASS] DD0016: is the LinuxKit VM running?
[PASS] DD0001: is the application running?
[SKIP] DD0018: does the host support virtualization?
[PASS] DD0002: does the bootloader have virtualization enabled?
[PASS] DD0017: can a VM be started?
[PASS] DD0024: is WSL installed?
[PASS] DD0021: is the WSL 2 Windows Feature enabled?
[PASS] DD0022: is the Virtual Machine Platform Windows Feature enabled?
[PASS] DD0025: are WSL distros installed?
[PASS] DD0026: is the WSL LxssManager service running?
[PASS] DD0029: is the WSL 2 Linux filesystem corrupt?
[PASS] DD0015: are the binary symlinks installed?
[PASS] DD0003: is the Docker CLI working?
[PASS] DD0013: is the $PATH ok?
[PASS] DD0005: is the user in the docker-users group?
[PASS] DD0007: is the backend responding?
[PASS] DD0014: are the backend processes running?
[PASS] DD0008: is the native API responding?
[PASS] DD0009: is the vpnkit API responding?
[PASS] DD0010: is the Docker API proxy responding?
[PASS] DD0006: is the Docker Desktop Service responding?
[PASS] DD0012: is the VM networking working?
[PASS] DD0032: do Docker networks overlap with host IPs?
[SKIP] DD0030: is the image access management authorized?
[PASS] DD0033: does the host have Internet access?
No fatal errors detected.
```

### Steps to reproduce the behavior

  1. Open the Docker Desktop window and leave it
  2. Open PowerShell and type `CTRL-R` to attempt to search shell history

**Comment by julianSelser** [comment]
+1

While it would be nice to only capture CTRL+R and others when the desktop app is focused, I'd also be happy with instructions on how to disable the feature entirely. Thanks.

**Comment by mat007** [comment]
Oh yes, that’s quite unfortunate. Sorry about that, we’re looking for a workaround.
We may have to do a patch release if we can’t find any.

Thanks for reporting it!

**Comment by mat007** [comment]
We’ve just released a fix for this bug in Desktop 4.4.3.
